### ISSUE TITLE
bninception 

### ISSUE THREAD

**Issue by Zhuysheng** [Cadene/pretrained-models.pytorch#87]
Epoch: [0][440/4769], lr: 0.00100       Time 0.231 (0.556)      Data 0.000 (0.154)      Loss nan (nan)  Prec@1 0.000 (0.454)    Prec@5 0.000 (2.268)
when train TSN with this bninception, loss become nan,but when i use bninception from [https://github.com/yjxiong/tensorflow-model-zoo.torch/tree/9788c674f3058741a4c4128142c21877052863ee](url),loss decrease,whats the problem

**Comment by Cadene** [comment]
Should be fixed: https://github.com/Cadene/pretrained-models.pytorch/commit/87e97515ec4480844e188d82e87e06376337a3af
